packages/hooks/src/useMutation.ts

```diff
@@ -5,10 +5,26 @@ import { DocumentNode } from 'graphql';
 import { MutationHookOptions, MutationTuple } from './types';
 import { MutationData } from './data/MutationData';
 
+let showApiChangeWarning = true;
+
 export function useMutation<TData = any, TVariables = OperationVariables>(
   mutation: DocumentNode,
   options?: MutationHookOptions<TData, TVariables>
 ): MutationTuple<TData, TVariables> {
+  // This is a temporary message intended to help lessen the blow of a
+  // breaking `useMutation` API change during the RA 3 beta process. It will
+  // be removed before the official launch of RA 3.
+  if (showApiChangeWarning) {
+    console.warn(
```

Comment:
Let’s use `invariant.warn` from `ts-invariant` to avoid increasing the bundle size.

Reply:
Arg - I keep forgetting about `invariant.warn` - thanks!